test/JDBC/expected/xml_exist-vu-verify.out

```diff
@@ -0,0 +1,459 @@
+
+
+DECLARE @xml NVARCHAR(100) = '<artists> <artist name="John Doe"/> <artist name="Edward Poe"/> <artist name="Mark The Great"/> </artists>'
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on nvarchar.)~~
+
+
+DECLARE @xml INT = 1
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on int.)~~
+
+
+DECLARE @xml BIGINT = 1
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on bigint.)~~
+
+
+DECLARE @xml FLOAT = 1
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on float.)~~
+
+
+DECLARE @xml DECIMAL = 1
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on decimal.)~~
+
+
+DECLARE @xml NUMERIC = 1
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on numeric.)~~
+
+
+DECLARE @xml BIT = 1
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on bit.)~~
+
+
+SELECT XmlColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_text
+GO
+~~START~~
+bit
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on text.)~~
+
+
+-- UDT on type VARCHAR
+SELECT VarUDTColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_udt
+GO
+~~START~~
+bit
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on varchar.)~~
+
+
+-- UDT on type IMAGE
+SELECT ImageUDTColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_udt
+GO
+~~START~~
+bit
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot call methods on image.)~~
+
+
+-- NULL values
+DECLARE @xml XML = NULL
+SELECT @xml.exist('/artists/artist/@name')
+GO
+~~START~~
+bit
+<NULL>
+~~END~~
+
+
+DECLARE @xml XML = '<artists> <artist name="John Doe"/> <artist name="Edward Poe"/> <artist name="Mark The Great"/> </artists>'
+SELECT @xml.exist(NULL)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The argument 1 of the XML data type method "exist" must be a string literal.)~~
+
+
+DECLARE @xml XML = NULL
+SELECT @xml.exist(NULL)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The argument 1 of the XML data type method "exist" must be a string literal.)~~
+
+
+-- column as input argument
+SELECT XmlColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_t1
+GO
+~~START~~
+bit
+1
+~~END~~
+
+
+SELECT babel_5222_xml_exist_t1.XmlColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_t1
+GO
+~~START~~
+bit
+1
+~~END~~
+
+
+SELECT dbo.babel_5222_xml_exist_t1.XmlColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Remote procedure/function reference with 4-part object name is not currently supported in Babelfish)~~
```

Comment:
In TSQL, calling exist function with syntax other than `col.query() or table.col.query()` will be matched to User defined functions, we tried to keep the same behaviour, but since four part object name is not currently supported in babelfish we are getting this error. Following is the output for same query in TSQL
```
SELECT dbo.babel_5222_xml_exist_t1.XmlColumn.exist('/artists/artist/@name') FROM babel_5222_xml_exist_t1
GO
~~ERROR (Code: 344)~~

~~ERROR (Message: Remote function reference 'dbo.babel_5222_xml_exist_t1.XmlColumn.exist' is not allowed, and the column name 'dbo' could not be found or is ambiguous.)~~

```